TombEngine/Game/control/trigger.cpp

```diff
@@ -132,17 +132,14 @@ int SwitchTrigger(short itemNumber, short timer)
 				item->Timer = FPS * timer;
 			return 1;
 		}
-		if (item->TriggerFlags != 6 || item->Animation.ActiveState)
+		if (item->TriggerFlags >= 0 || item->Animation.ActiveState)
```

Comment:
is `item->TriggerFlags` ever less than 0?

Reply:
This is my concern.

TriggerFlags shall never be less than 0. But, if I remove that condition, then it only remains if (ActveState), in such situation, the code doesn't always enter and in the practice, the switches can't switched off.

item->TriggerFlags != 6 was meaning that the thread always enters except when ocb is 6 that in TEN is for the wall hole. I had to remove the wall hole restriction, but I couldn't remove the condition totally, so I put a value that won't cause problem, a negative one.